scripts/common.sh.in

```diff
@@ -371,3 +371,28 @@ check_minion_id() {
         return 1
     fi
 }
+
+rotate_logfile() {
+    # because set -u, we pass an empty string as default value
```

Comment:
Could have been handled directly in the function when retrieving the variable but I'm fine with this approach